src/atoms/FlashMessage/Readme.md

```diff
@@ -0,0 +1,20 @@
+Transition Example:
+
+```example
+  <div>
+    <Button onClick={() => setState({ showMessage: !state.showMessage })}>Click me!</Button>
+    <Spacer xsmall />
+    { state.showMessage &&
+      <FlashMessage variant='fadeInDown'>
+        <ImageAside
+          small
+          basic={false}
```

Comment:
@patrickkim We shouldn't have to do this. I made a ticket to refactor this: https://app.clubhouse.io/policygenius/story/2216/refactor-image-aside